ROMFS/px4fmu_common/init.d-posix/1035_plane_ctrlalloc

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+#
+# @name Plane SITL
+#
+
+param set CA_AIRFRAME 3
+
+sh /etc/init.d/rc.fw_defaults
+sh /etc/init.d/rc.ctrlalloc
+
+if [ $AUTOCNF = yes ]
+then
+	param set EKF2_ARSP_THR 8
+	param set EKF2_FUSE_BETA 1
+	param set EKF2_MAG_ACCLIM 0
+	param set EKF2_MAG_YAWLIM 0
+
+	param set FW_LND_AIRSPD_SC 1
+	param set FW_LND_ANG 8
+	param set FW_THR_LND_MAX 0
+
+	param set FW_L1_PERIOD 15
+
+	param set FW_P_TC 0.5
+	param set FW_PR_FF 0.40
+	param set FW_PR_I 0.05
+	param set FW_PR_P 0.05
+
+	param set FW_R_TC 0.7
+	param set FW_RR_FF 0.20
+	param set FW_RR_I 0.02
+	param set FW_RR_P 0.22
```

Comment:
Is the FW rate controller used at all?

Reply:
@jlecoeur No it's not, this is just a copy past artifact. It would be great to write an attitude_control module which can generate rate setpoints for both mc and fw. Then we don't need to use the old modules any more.